README.md

```diff
@@ -69,7 +70,7 @@ func main() {
 		Destinations: []walletrpc.Destination{
 			{
 				Address: "45eoXYNHC4LcL2Hh42T9FMPTmZHyDEwDbgfBEuNj3RZUek8A4og4KiCfVL6ZmvHBfCALnggWtHH7QHF8426yRayLQq7MLf5",
-				Amount:  10000000000, // 0.01 XMR
+				Amount:  10*unit.Millinero, // 0.01 XMR
```

Comment:
shouldn't it be Centinero?

Reply:
Yes, ideally one'd use `1 Centinero` here, but I preferred `10 Millinero`. `Millinero` — especially in example code — sounds much better imo. Should I change it to `1 Centinero`?